assets/i18n/strings_hu.i18n.json

```diff
@@ -169,7 +169,7 @@
     "title": "Változásnapló"
   },
   "aliasGenerator(ignoreMissing)": {
-    "@info": "Inherits from the English version"
+    "@info": "Az angol változatból örökölte"
```

Comment:
Comments should not be translated

Reply:
Ok. Did you fix it or should I do it and add another pull request?

Reply:
This is not a major issue, so I just merge it :)